devtools/server/actors/replay/auth.js

```diff
@@ -125,11 +138,8 @@ function handleAuthChannelMessage(_id, message, target) {
   // (the app) to request a login which would launch the sign in page in the
   // user's preferred browser.
   if (type === "connect") {
-    deferredAccessToken.promise.then(token => {
-      if (authChannel) {
-        authChannel.send({ token }, target);
```

Comment:
I'm actually a bit surprised here about how this was working previously, because `registerWebChannel` assigns this, except we call `registerWebChannel` twice and the second one is with `localUrl` which means this was probably always using the wrong `authChannel` (with the right `target`).

We should adjust this so that `handleAuthChannelMessage` knows what channel it is listening to, and we need to make sure we always use that channel to reply to the `connect` message?